implementations/rust/ockam/ockam_api/src/ui/terminal/mod.rs

```diff
@@ -219,37 +229,37 @@ impl<W: TerminalWriter + Debug> Terminal<W, ToStdErr> {
         self.stderr.is_tty()
     }
 
+    /// Return true if log messages are emitted to the console
+    fn logging_to_console(&self) -> bool {
+        self.logging_enabled && !self.logging_goes_to_file
+    }
+
+    /// Return true if we can write to stderr
+    /// We can write to stdout unless all the message are logged to the console
```

Comment:
I found this line here a little confusing because it mentions the stdout and we are in the `ToStdErr` methods. Maybe we could omit it?

Reply:
I fixed the comment